### ISSUE TITLE
Movie parsed as episode

### ISSUE THREAD

**Issue by l3uddz** [guessit-io/guessit#465]
Input:
/mnt/plexdrive/Media/Movies/Open Season 2 (2008)/Open Season 2 (2008) - Bluray-1080p.x264.DTS.mkv

Output:

MatchesDict([('title', 'Open'), ('season', 2), ('year', 2008), ('format', 'BluRay'), ('screen_size', '1080p'), ('video_codec', 'h264'), ('audio_codec', 'DTS'), ('container', 'mkv'), ('mimetype', 'video/x-matroska'), ('type', 'episode')])

using guessit 2.1.4 - have tried with and without the full path, same result.

**Comment by ratoaq2** [comment]
Is it an option for you to use `--type movie` when calling guessit?

**Comment by l3uddz** [comment]
I actually use guessit to traverse a media folder looking for dupes based on a hash of the result from guessit. I can definitely use this flag as currently it uses the type returned from guessit to determine if it's a movie or episode. I guess it would be nicer not too have todo this, but I appreciate it may not be possible to determine this as a movie without breaking other stuff.

**Comment by ratoaq2** [comment]
This one is really hard to handle.
`Open Season 2 (2008)` is a movie
but
`Breaking Bad Season 1 (2008)` is not a movie

For such corner case scenarios you will need to try some properties to help guessit on decision making.

Another possibility is to pass `-T "Open Season 2" to guessit parameter. All parameters passed as `-T` will be part of a list of known titles that guessit will use when parsing names.

**Comment by l3uddz** [comment]
I'll make some changes to my script as I did figure it would be pretty hard to parse correctly due to having season in the name.  Will take a look at the --T option thanks.
